Add table of noise types for each column (#151)
ihmeuw/pseudopeople: docs/source/noise/index.rst

Default Noise Types for Each Column
-----------------------------------

.. list-table:: Types of noise for each column
  :widths: 20 20 20 20
  :header-rows: 0

  * - Column name
    - Datasets present
    - Types of noise
    - Notes
  * - First name
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
    - Leave a field blank, use a fake name, make typos
    -
  * - Middle initial
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
    - Leave a field blank, make typos
    -
  * - Last name
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
    - Leave a field blank, use a fake name, make typos

>     - The list of fake names will be different than the first names

Reviewer: ```suggestion
    -
```

Ah sorry can we remove this too? I think it is confusing and already explained better in the details section.

    - Decennial Census, ACS, CPS, WIC, W-2 and 1099
    - Leave a field blank, write the wrong digits, make typos
    - Noise for all types of addresses works in the same way
  * - Street name for any address (Home OR Mailing OR Employer)
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099
    - Leave a field blank, make typos
    - Noise for all types of addresses works in the same way
  * - Unit number for any address (Home OR Mailing OR Employer)
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099
    - Leave a field blank, write the wrong digits, make typos
    - Noise for all types of addresses works in the same way
  * - PO Box for mailing address
    - W-2 and 1099
    - Leave a field blank, write the wrong digits, make typos
    -
  * - City name for any address (physical, mailing, or employer)
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099
    - Leave a field blank, make typos
    - Noise for all types of addresses works in the same way
  * - State for any address (physical, mailing, or employer)
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099
    - Leave a field blank 
    - Noise for all types of addresses works in the same way

>   * - Zip code for any address (physical, mailing, or employer)

Reviewer: ```suggestion
  * - ZIP code for any address (physical, mailing, or employer)
```

Default Noise Types for Each Column
-----------------------------------

.. list-table:: Types of noise for each column
  :widths: 20 20 20 20
  :header-rows: 0

>   * - Attribute name

Reviewer: ```suggestion
  * - Column name
```

    - Datasets present
    - Types of noise
    - Notes
  * - First name

>     - Census, Household Surveys, WIC, Taxes (both), SSA

Reviewer: Can you match the Dataset names to what they are called on the Datasets page, using abbreviations? And remove 1040 since that is not implemented. E.g. this line would be

```suggestion
    - Decennial Census, ACS, CPS, WIC, W-2 and 1099, SSA
```

    - Leave a field blank, use a fake name, make typos
    -
  * - Middle initial
    - Census, Household Surveys, WIC, Taxes (both), SSA
    - Leave a field blank, make typos
    -
  * - Last name
    - Census, Household Surveys, WIC, Taxes (both), SSA
    - Leave a field blank, use a fake name, make typos
    - The list of fake names will be different than the first names
  * - Age
    - Census, Household Surveys, Taxes (both)

>     - Leave a field blank, Misreport age, make typos

Reviewer: ```suggestion
    - Leave a field blank, misreport age, make typos
```

    -
  * - Date of birth
    - Census, Household Surveys, WIC, Taxes (both), SSA
    - Leave a field blank, write the wrong digits, make typos
    -

>   * - Street number for any address (Home OR Mailing OR Employer)

Reviewer: ```suggestion
  * - Street number for any address (physical, mailing, or employer)
```
and in the other rows with this.

    - Census, Household Surveys, WIC, Taxes (both)
    - Leave a field blank, write the wrong digits, make typos

>     - Noise for all types of addresses will work in the same way

Reviewer: ```suggestion
    - Noise for all types of addresses works in the same way
```
and in the other rows with this.

    - Noise for all types of addresses will work in the same way
  * - PO Box for mailing address
    - Taxes (both)
    - Leave a field blank, write the wrong digits, make typos
    -
  * - City name for any address (Home OR Mailing OR Employer)
    - Census, Household Surveys, WIC, Taxes (both)
    - Leave a field blank, make typos
    - Noise for all types of addresses will work in the same way
  * - State for any address (Home OR Mailing OR Employer)
    - Census, Household Surveys, WIC, Taxes (both)
    - Leave a field blank 
    - Noise for all types of addresses will work in the same way
  * - Zip code for any address (Home OR Mailing OR Employer)
    - Census, Household Surveys, WIC, Taxes (both)
    - Leave a field blank, write the wrong zipcode digits, make typos
    -
  * - Relationship to head of household
    - Census
    - Leave a field blank
    -
  * - Sex
    - Census, Household Surveys, WIC
    - Leave a field blank
    -
  * - Race/ethnicity
    - Census, Household Surveys, WIC
    - Leave a field blank
    -
  * - SSN
    - Taxes (both), SSA
    - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos

>     - Note that not all types of noise apply to all datasets

Reviewer: I don't think this is true (anymore?). Let's delete this note.

Reviewer: Ah, I get it -- I think it's just about borrowed SSN. So:
```suggestion
    - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset
```

>   * - ITIN

Reviewer: We can remove ITIN entirely, I think, since 1040 is not implemented.
